src/third_party/CMakeLists.txt

```diff
@@ -96,11 +96,11 @@ if (DEFINED OpenMVG_USE_INTERNAL_EIGEN)
   add_subdirectory(eigen)
 endif()
 
-list(APPEND directories cmdLine histogram htmlDoc progress vectorGraphics)
+list(APPEND directories cmdLine histogram htmlDoc progress vectorGraphics hnswlib)
 foreach(inDirectory ${directories})
 install(
   DIRECTORY ./${inDirectory}
-  DESTINATION include/openMVG/third_party/
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/include/openMVG/third_party/
```

Comment:
Not sure we need this... Cmake should use it by default if CMAKE_INSTALL_PREFIX is defined.

Reply:
yes I just checked in the doc, you are right